lang/nl/lang.php

```diff
@@ -36,7 +36,7 @@
         'clear_cache_link' => 'Leeg cache',
         'clear_cache_loading' => 'Applicatie cache legen...',
         'clear_cache_success' => 'De applicatie cache is succesvol geleegd.',
-        'clear_cache_hint' => 'Het is verstandig om regelmatig op <strong>Leeg cache</strong> te klikken om de veranderingen te zien op de website.',
+        'clear_cache_hint' => 'Het is verstandig om regelmatig op <strong>Leeg cache</strong> te klikken om de wijzigingen te zien op de website.',
```

Comment:
This isn't really inline with original english string. 
```You may need to click <strong>Clear cache</strong> to see the changes on the front-end.'```

The literal translation of the current text is: "It may be wise".

Personally I would go for something like this:
``` Het kan zijn dat het nodig is om op "cache legen" te klikken om wijzigingen op de website te zien.```

Reply:
Agreed

Reply:
Not to be picky, but where's the <strong> tag?

Reply:
And L29 ("hint_locales") has a trailing space.

Reply:
Changed in #58f4de5